ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml

```diff
@@ -104,6 +102,19 @@
     #TODO: rpm used in place of yum or rpm_key module
     - skip_ansible_lint
 
+- name: Sed change gpgcheck for SLES12 on x86_64
```

Comment:
Why is this needed - if we have to do this instead of doing a proper GPG check then it needs a comment explaining why

Reply:
I'm not sure- It was part of the `Enable NVidia_Cuda_Toolkit for SLES x86_64` tasks, I just split it up as `rpm` was reading in every word as a package to install.

Reply:
Can you do some experimentation and see if it's still needed and if so what's the message - we should document it in this playbook